templates/section_pages/support_pages.hbs

```diff
@@ -61,9 +61,9 @@
                         </a>
                     {{/is}}
                     {{#is id 1500002273242}} {{!-- User Forums --}}
-                        <a href="https://www.configura.com/my/forum" class="custom-card-link">
+                        <a href="https://www.configura.com/my/forum" target="_blank" class="custom-card-link">
```

Comment:
@dinka01  If you want the page to open in a new tam you also need to add `rel="noopener noreferrer"` to the <a href/>

Reply:
@abergwall Done.